base/applications/rapps/lang/pl-PL.rc

```diff
@@ -202,7 +202,7 @@ BEGIN
     IDS_APPLICATIONS "Aplikacje"
     IDS_CHOOSE_FOLDER_TEXT "Wybierz katalog, w którym będa zapisywane pobrane programy:"
     IDS_CHOOSE_FOLDER_ERROR "Wybrałeś nieistniejący katalog! Czy chcesz utworzyć nowy?"
-    IDS_URL_INVALID "The URL you have specified is invalid or not supported. Please correct it!"
+    IDS_URL_INVALID "Określony adres URL jest nieprawidłowy lub nieobsługiwany. Popraw go."
```

Comment:
```suggestion
    IDS_URL_INVALID "Określony adres URL jest nieprawidłowy lub nieobsługiwany. Popraw go!"
```
?

Reply:
I don't know. It kinda "sounds" rude to me as a exclamation, and replacing it with dot doesn't really change the meaning anyway, so I thought it would be better with dot.

Reply:
OK, nothing to do here